### PR TITLE
Racial Skills

### PR DESCRIPTION
**Pull request by servusdei2018**
I modified the skills per the requested:

```
Felinus -> Clawing
Lobohian -> Bite, Smell
Dragon -> Clawing, Bite, Smell, Wings
```

### CODE REVIEW

File: src/h/mud.h [hunk 1]
```diff
@@ -575,7 +575,7 @@ typedef enum
   RACE_HUMAN, RACE_ELF, RACE_DWARF, RACE_HALFLING, RACE_PIXIE, RACE_OGRE,
   RACE_ORC, RACE_TROLL, RACE_SHADE, RACE_GOBLIN, RACE_DROW, RACE_GNOME,
   RACE_CENTAUR, RACE_DRAGON, RACE_CELESTIAL, RACE_VAMPIRE, RACE_DEMON,
-  RACE_MINDFLAYER, RACE_ANIMAL
+  RACE_MINDFLAYER, RACE_ANIMAL, RACE_FELINUS, RACE_LOBOHIAN
```
Comment (servusdei2018) [inline src/h/mud.h:578]: Two race constants were added.

File: src/h/mud.h [hunk 2]
```diff
@@ -598,6 +598,8 @@ typedef enum
 #define RACE_DEMON          16
 #define RACE_MINDFLAYER     17
 #define RACE_ANIMAL         18
+#define RACE_FELINUS        19
+#define RACE_LOBOHIAN       20
```
Comment (servusdei2018) [inline src/h/mud.h:602]: Two new race macros were defined.

File: src/skills.c [hunk 3]
```diff
@@ -3768,6 +3768,12 @@ void do_bite(CHAR_DATA *ch, char *argument)
     send_to_char("Your new form prevents the use of this skill.\r\n", ch);
     return;
   }
+  
+   // Restrict the bite ability to the Lobohian and Dragon races.
+  if(ch->race != RACE_LOBOHIAN && ch->race != RACE_DRAGON) {
+    send_to_char("That isn't quite one of your natural skills.\r\n", ch);
+    return;
+  }
```
Comment (servusdei2018) [inline src/skills.c:3776]: Bite was restricted to Lobohians and Dragons

File: src/skills.c [hunk 4]
```diff
@@ -3956,6 +3962,12 @@ void do_claw(CHAR_DATA *ch, char *argument)
     send_to_char("That isn't quite one of your natural skills.\r\n", ch);
     return;
   }
+  
+  // Restrict the claw ability to the Felinus and Dragon race.
+  if(ch->race != RACE_FELINUS && ch->race != RACE_DRAGON) {
+    send_to_char("That isn't quite one of your natural skills.\r\n", ch);
+    return;
+  }
```
Comment (servusdei2018) [inline src/skills.c:3970]: Claw was restricted to Felinus and Dragons.

**Comment by varchild** [comment]
Start racial skills

**Pull request merged**